src/calendar/day/basic/index.js

```diff
@@ -95,14 +85,16 @@ class Day extends Component {
       <TouchableOpacity
         testID={this.props.testID}
         style={containerStyle}
-        onPress={this.onDayPress}
+        onPress={!shouldDisableTouchEvent ? this.onDayPress : undefined}
```

Comment:
Usually I don't like changing production code to satisfy tests, but it looked like the easiest option in this case.

Reply:
I don't mind that. Then the same should go to 'onLongPress'

Reply:
Done